src/main/java/mate/academy/spring/controller/UserController.java

```diff
@@ -0,0 +1,48 @@
+package mate.academy.spring.controller;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import mate.academy.spring.dto.UserResponseDto;
+import mate.academy.spring.mapper.UserDtoMapper;
+import mate.academy.spring.model.User;
+import mate.academy.spring.service.UserService;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/users")
+public class UserController {
+    private final UserService userService;
+    private final UserDtoMapper userDtoMapper;
+
+    public UserController(UserService userService, UserDtoMapper userDtoMapper) {
+        this.userService = userService;
+        this.userDtoMapper = userDtoMapper;
+    }
+
+    @GetMapping("/inject")
+    public String inject() {
+        User bobby = new User("Bobby", "Alison");
+        userService.add(bobby);
+        User ben = new User("Ben", "Geller");
+        userService.add(ben);
+        User emma = new User("Emma", "Green");
+        userService.add(emma);
+        return "Injecting users was successful!";
+    }
+
+    @GetMapping("/")
+    public List<UserResponseDto> getAll() {
+        return userService.listUsers()
+                .stream()
+                .map(userDtoMapper::parseToDto)
+                .collect(Collectors.toList());
+    }
+
+    @GetMapping("/{id")
```

Comment:
Цей код працює? Можливо це має бути так?
```suggestion
    @GetMapping("/{id}")
```